core/store/models/log_events.go

```diff
@@ -254,6 +255,19 @@ func (le InitiatorLogEvent) JSON() (JSON, error) {
 	return out, json.Unmarshal(b, &out)
 }
 
+// Meta returns metadata about the eth log
```

Comment:
Maybe the full term 'ethereum' could be better than the shorthand for the comment.  "eth" is also used to reference ethernet devices.